modules/manage/partials/remote-read-replicas.adoc

```diff
@@ -36,11 +33,12 @@ endif::[]
 
 You need the following:
 
-* An origin cluster with xref:{tiered-storage-link}#set-up-tiered-storage[Tiered Storage] set up.
+* An origin cluster with xref:{tiered-storage-link}#set-up-tiered-storage[Tiered Storage] set up. Multi-region buckets or containers are not supported.
 * A topic on the origin cluster, which you can use as a Remote Read Replica topic on the remote cluster.
-* A separate remote cluster in the same region as the bucket or container used for the origin cluster.
-** If you use a multi-region bucket/container, you can create the read replica cluster in any region that has that bucket/container.
-** If you use a single-region bucket/container, the remote cluster must be in the same region as the bucket/container.
+* A separate remote cluster. 
+** AWS: The remote cluster must be in the same region as the origin cluster's storage bucket/container.
```

Comment:
So sorry, these 3 bits of information are applicable to Cloud. I can't speak to the self-managed side. Sorry I wasn't paying attention and steered you wrong. :(